101-azure-bastion/azuredeploy.parameters.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "vnet-name": {
+      "value": "vnet01"
+    },
+    "vnet-ip-prefix": {
+      "value": "10.1.0.0/16"
+    },
+    "vnet-new-or-existing": {
+      "value": "new"
+    },
+    "bastion-subnet-ip-prefix": {
+      "value": "10.1.1.0/27"
+    },
+    "bastion-host-name": {
+      "value": "GEN-UNIQUE-8"
+    },
+    "location": {
+      "value": "southcentralus"
```

Comment:
is the preview only available in certain regions?

Reply:
Yes, the preview is currently only available in a subset of regions, but is in the process of rolling out to more regions.  By GA, all regions will be supported.